Doc/library/dis.rst

```diff
@@ -382,6 +382,10 @@ The Python compiler currently generates the following bytecode instructions.
 
 **General instructions**
 
+In the following, TOS(i) refer to the *i*-th item on the stack which is index from 1.
+We also use as shorthand TOS = TOS(1) which is the top-of-stack.
+TOS1, TOS2, TOS3 are the second, third and fourth items on the stack, respectively.
```

Comment:
It's a bit unfortunate that TOS1 = TOS(2),   TOS2 = TOS(3), etc. 

@markshannon - would you like to change this?

Reply:
This was another reason for my previous formulation that avoided ro shed too much light on this.

Reply:
Generally we count from one for stack items, so I would like to change it. Ideally drop `TOS1` etc, all together.
The current explanation is not explicit about what is popped or not, so the shorthand is not helpful.